crates/sage-cli/src/input.rs

```diff
@@ -39,6 +40,9 @@ pub struct Search {
 
     #[serde(skip_serializing)]
     pub annotate_matches: bool,
+
+    #[serde(skip_serializing)]
```

Comment:
Do not skip serializing. Sage results.json files should contain all information required to reproduce a search (and can be used as inputs to do so).

Reply:
Removed #[serde(skip_serializing)].